_internal/contracts/errors-contract/Makefile

```
@@ -0,0 +1,38 @@
+	stellar contract deploy \
+		--wasm $(WASM_PATH) \
+		--network $(NETWORK) \
+		--source-account $(SOURCE_ACCOUNT)
+
+bindings:
+	stellar contract bindings typescript \
+		--wasm $(WASM_PATH) \
+		--output-dir $(BINDINGS_DIR) \
+		--overwrite
+
+clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)